packages/manager/modules/pci/src/projects/project/failover-ips/imports/imports.controller.js

```diff
@@ -0,0 +1,41 @@
+import filter from 'lodash/filter';
+import isEmpty from 'lodash/isEmpty';
+import isObject from 'lodash/isObject';
+import map from 'lodash/map';
+
+export default class FailoverIpImportsController {
+  /* @ngInject */
+  constructor($q, OvhApiIp, projectId) {
+    this.$q = $q;
+    this.OvhApiIp = OvhApiIp;
+    this.projectId = projectId;
+    this.ips = null;
+
+    this.loadData();
+  }
+
+  loadData() {
+    return this.OvhApiIp
+      .v6()
+      .query({
+        type: 'failover',
+      })
+      .$promise
+      .then(ips => this.$q.all(
+        map(
+          ips,
+          ip => this.OvhApiIp
+            .v6()
+            .get({
+              ip,
+            })
+            .$promise,
+        ),
+      ))
+      .then(ips => filter(ips, ({ routedTo }) => isEmpty(routedTo)
+        || (isObject(routedTo) && routedTo.serviceName !== this.projectId)))
```

Comment:
```suggestion
        || get(routedTo, serviceName) !== this.projectId))
```
?

Reply:
I think is better to check if `routedTo` is an object :)